### DISCUSSION TITLE
Get event stats (GET /api/v1/admin/event/:slug/stats)

### DISCUSSION THREAD

**Discussion by cappe-responsiva** [alfio-event/alf.io#1063] [Q&A]
Hi,

looking back to the DRAFT PUBLIC API (https://github.com/alfio-event/alf.io/issues/390) there is a get method for retriving stats of an event (for example tickets available). Is it still working? Is no more mentioned in the current Wiki.

I did a test using the GET /api/v1/admin/event/:slug/stats but I got an error: "unexpected exception". 
I only use the slug of the event in the string and try to get the informations with PHP.

Best,
Claudio

**Comment by cbellone** [comment]
You're right, we didn't include that API in the official documentation. Thanks for letting us know.

I can confirm it's available. We'll update the doc as soon as possible.

Please make sure you're including a valid API Key in your request request, see

```
curl -H "Authorization: ApiKey 2306e3c7-363d-48b3-bdfe-48884900f2b6" http://localhost:8080/api/v1/admin/event/tap/stats
```
